packages/ui/v2/core/Shell.tsx

```diff
@@ -780,9 +781,10 @@ function SideBar() {
           <Navigation />
         </div>
 
-        {isCalcom && <Tips />}
-
-        <div className="fixed left-1 bottom-0 w-4 bg-gray-50 before:absolute before:left-0 before:-top-20 before:h-20 before:w-48 before:bg-gradient-to-t before:from-gray-50 before:to-transparent lg:left-4 lg:w-48">
+        <div>
+          <LicenseRequired toHide>
```

Comment:
This now shows a enterprise error when viewed on a platform without enterprise key. That's not desired I think? 

![CleanShot 2023-01-02 at 09 30 01@2x](https://user-images.githubusercontent.com/2969573/210208489-4513b38c-db9f-4616-a615-b183f39a8b73.jpg)

Reply:
oh right good point. it should show the children without License. (if toHide is true)

Reply:
can you check if this was resolved? with my latest commit

Reply:
https://github.com/calcom/cal.com/pull/6228/commits/f65422b0e188202ecb6d9bdc9ffbe00193803505